book/src/chap01.asciidoc

```diff
@@ -182,9 +182,9 @@ Ambigüedad::
 Los lenguajes naturales están llenos de ambigüedades, este problema es abordado mediante el uso del contexto y otra información. Los lenguajes formales están diseñados para ser casi completamente precisos, lo que significa que cualquier declaración tiene exactamente un significado, independientemente del contexto.
 
 Redundancia::
-Para compensar la ambigüedad y reducir los malentendidos, los lenguajes naturales emplean mucha redundancia. Como resultado, a menudo hace un uso excesivo de palabras. Los lenguajes formales son menos redundantes y más concisos.
+Para compensar la ambigüedad y reducir los malentendidos, los lenguajes naturales emplean mucha redundancia. Como resultado, a menudo son verboso. Los lenguajes formales son menos redundantes y más concisos.
```

Comment:
son verbosos instead of son verboso

Reply:
doesn't sound right to me

Reply:
i dont like it either, but i didnt found any good traslation at the time